cardano-api/src/Cardano/Api/Typed.hs

```diff
@@ -1565,6 +1564,103 @@ data MultiSigScript = RequireSignature (Hash PaymentKey)
                     | RequireMOf Int [MultiSigScript]
   deriving (Eq, Show)
 
+instance ToJSON MultiSigScript where
+  toJSON (RequireSignature payKeyHash) = String . Text.decodeUtf8 $ serialiseToRawBytesHex $ payKeyHash
+  toJSON (RequireAnyOf reqSigs) = object [ "any" .= map toJSON reqSigs ]
+  toJSON (RequireAllOf reqSigs) = object [ "all" .= map toJSON reqSigs ]
+  toJSON (RequireMOf reqNum reqSigs) =
+    case reqNum > length reqSigs of
+      False -> object [convertMofN .= map toJSON reqSigs]
+      True -> error $ "mofn: Required number of signatures \
+                      \exceed number of payment key hashes provided. m: "
+                      ++ show reqNum ++ " n: " ++ show (length reqSigs)
+   where
+     convertMofN = Text.pack $ (show reqNum ++ "of" ++ show (length reqSigs))
```

Comment:
To stick with the `camelCase` convention for JSON keys, I would use `Of`:

```suggestion
     convertMofN = Text.pack $ (show reqNum ++ "Of" ++ show (length reqSigs))
```

So now we'd have keys like `3Of5` and `2Of6`